frontend/src/components/dashboard/aliases/Alias.module.scss

```diff
@@ -0,0 +1,296 @@
+    background-color: rgba($color-white, 0.5);
+  }
+
+  .controls {
+    flex-grow: 2;
+  }
+}
+
+.mainData {
+  display: flex;
+  padding: $spacing-md 0;
+}
+
+.controls {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  row-gap: $spacing-xs;
+  align-items: center;
+
+  // The label field might not be present,
+  // but if it is (i.e. the copy button/email
+  // address is the third element),
+  // the copy button should be in the same column,
+  // one row below.
+  *:nth-child(3) {
+    grid-column: 2;
+  }
+}
+
+.toggleButton {
+  cursor: pointer;
+  height: 1.15rem;
+  width: 2rem;
+  position: relative;
+  overflow: hidden;
+  border: none;
+  border-radius: 1.5rem;
+  margin-right: $spacing-sm;
+  outline: none;
+  background-color: $color-blue-50;
+  background-size: 20px;
+
+  &:hover,
+  &:focus {
+    background-color: $color-blue-70;
+    box-shadow: $buttonFocusOutline;
+  }
+
+  &:active {
+    background-color: $color-blue-80;
+  }
+
+  // The knob:
+  &::after {
+    content: "";
+    height: 0.75rem;
+    width: 0.75rem;
+    position: absolute;
+    border-radius: 50%;
+    background-color: $color-white;
+    top: 0;
+    bottom: 0;
+    margin: auto;
+    right: 4px;
+    transition: left $toggleTransitionDuration ease,
+      right $toggleTransitionDuration ease,
+      background-color $toggleTransitionDuration ease;
+  }
+
+  .isDisabled & {
+    background-color: $color-light-gray-30;
+
+    &:hover,
+    &:focus {
+      background-color: $color-light-gray-40;
+    }
+
+    &:active {
+      background-color: $color-light-gray-50;
+    }
+
+    &::after {
+      left: 4px;
+      right: 18px;
+    }
+  }
+}
+
+.copyControls {
+  display: flex;
+  align-items: center;
+}
+
+.copyButtonWrapper {
+  position: relative;
+
+  .copyButton {
+    @include text-body-lg;
+    display: flex;
+    align-items: center;
+    background-color: transparent;
+    border: 0px none transparent;
+    cursor: pointer;
+    padding: 0 $spacing-sm;
+
+    .address {
+      font-family: $font-stack-firefox;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      max-width: $content-md;
+    }
+
+    .copyIcon {
+      padding: 0 $spacing-sm;
+      opacity: 0.5;
+    }
+
+    &:hover {
+      .address {
+        text-decoration: underline;
+      }
+
+      .copyIcon {
+        opacity: 1;
+      }
+    }
+  }
+
+  .copiedConfirmation {
+    opacity: 0;
+    pointer-events: none;
+    transition: opacity 2s;
+    background-color: $color-violet-30;
+    color: $color-white;
+    border-radius: $border-radius-md;
+    padding: $spacing-xs;
+    // By allowing this to overlap other elements,
+    // we don't need to reserve empty space for it.
+    // Otherwise, this empty space would push the
+    // .expandToggle out of the card on small screens:
+    position: absolute;
+    left: 50%;
+    top: 0;
+    // right: 0;
+    transform: translateX(-50%) translateY(-100%);
+
+    &.isShown {
+      pointer-events: auto;
```

Comment:
Oh yeah, I think I went with showing the tooltip _above_ the copied address to make it fit better on small screens, rather than showing it to the right like the current website does (in which the tooltip isn't visible at all on small screens). I think the new behaviour is the least of both evils...

Reply:
Makes sense!